src/components/pages/Account.tsx

```diff
@@ -20,32 +28,69 @@ type DispatchProps = {
 
 type StateProps = {
   loggedInMember: Member;
+  trustedForRecovery: Member;
+  votingFor: Member;
+};
+
+type Props = DispatchProps & OwnProps & StateProps;
+
+type State = {
+  timeRemaining: string;
 };
 
-type Props = DispatchProps & OwnProps;
+const Break: React.StatelessComponent<{}> = () => <View style={styles.break} />;
 
 // TODO show when account will go inactive, total Raha outstanding,
 // leaderboard, who you are voting for, etc.
 // Eventually it could be AccountTab instead of ProfileTab.
-const AccountView: React.StatelessComponent<Props> = props => (
-  <View style={styles.container}>
-    <Button title="Sign Out" onPress={props.signOut} />
-  </View>
-);
+class AccountView extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      timeRemaining: this.getTimeRemaining()
+    };
+  }
+
+  private getTimeRemaining() {
+    // TODO make a RED countdown with finer granularity than 1 day
+    // if you're within a month of being marked inactive, call
+    // updateTimeRemaining on setInterval.
+    const sinceLastMintMilli =
+      Date.now() - this.props.loggedInMember.lastMinted.getTime();
+    return `${400 - Math.round(sinceLastMintMilli / MS_PER_DAY)} days`;
```

Comment:
Shouldn't the timer be since last transaction instead of last minted?

Reply:
Yes, created https://github.com/rahafoundation/raha-mobile-app/issues/153